nautobot/circuits/forms.py

```diff
@@ -65,7 +65,7 @@ class Meta:
 class ProviderCSVForm(CustomFieldModelCSVForm):
     class Meta:
         model = Provider
-        fields = Provider.csv_headers
+        fields = []  # TODO
```

Comment:
These empty fields are for CSV import right?

Reply:
Right, I'm *hoping* that the `*CSVForm` classes will be able to go away entirely in #3715. Haven't gotten there yet. :-)